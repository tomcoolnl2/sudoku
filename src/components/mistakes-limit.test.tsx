
import { render } from '@testing-library/react'
import { withTheme, withRedux } from '../test'
import { MistakesLimit } from '.'

describe('MistakesLimit', () => {
	
	it('should match it\'s snapshot', () => {
		const { container } = render(withTheme(withRedux(<MistakesLimit />)))
		expect(container.firstChild).toMatchSnapshot()
	})
})