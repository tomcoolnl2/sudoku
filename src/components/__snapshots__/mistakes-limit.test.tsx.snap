// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MistakesLimit should match it's snapshot 1`] = `
<label
  for="mistakes-limit"
>
  Allowed Mistakes:
</label>
`;

exports[`mistakesLimit should match it's snapshot 1`] = `
<label
  for="mistakes-limit"
>
  Allowed Mistakes:
</label>
`;
